--- src/dbStatic/dbYacc.y	2018-09-14 20:44:05.000000000 +0200
+++ src/dbStatic/dbYacc.y	2026-08-24 19:38:35.756802000 +0200
@@ -7,7 +7,7 @@
 * in file LICENSE that is included with this distribution. 
 \*************************************************************************/
 %{
-static int yyerror();
+static int yyerror(char *);
 static int yy_start;
 static long pvt_yy_parse(void);
 static int yyFailed = 0;
--- src/dbStatic/dbStaticRun.c	2018-09-14 20:44:05.000000000 +0200
+++ src/dbStatic/dbStaticRun.c	2026-08-24 19:48:16.003737000 +0200
@@ -177,7 +177,7 @@
 
     if (pdset->init) {
         pthisDevSup = pdevSup;
-        pdset->init(0);
+        pdset->init();
         pthisDevSup = NULL;
     }
 }

