--- src/toolsComm/flex/ccl.c	2018-09-14 20:44:05.000000000 +0200
+++ src/toolsComm/flex/ccl.c	2026-08-24 18:55:03.766937000 +0200
@@ -140,7 +140,7 @@
 void list_character_set(FILE *file, int cset[])
 {
     int i;
-    char *readable_form();
+    char *readable_form(int);
 
     putc( '[', file );
 
--- src/toolsComm/flex/gen.c	2018-09-14 20:44:05.000000000 +0200
+++ src/toolsComm/flex/gen.c	2026-08-24 19:03:55.437935000 +0200
@@ -225,7 +225,7 @@
     int i, j;
     static char C_char_decl[] = "static const %s %s[%d] =\n    {   0,\n";
     int numrows;
-    Char clower();
+    Char clower(int);
 
     if ( numecs < csize )
 	printf( C_char_decl, "YY_CHAR", "yy_ec", csize );
@@ -245,7 +245,7 @@
 
     if ( trace )
 	{
-	char *readable_form();
+	char *readable_form(int);
 
 	fputs( "\n\nEquivalence Classes:\n\n", stderr );
 
--- src/toolsComm/flex/main.c	2018-09-14 20:44:05.000000000 +0200
+++ src/toolsComm/flex/main.c	2026-08-24 19:21:30.905847000 +0200
@@ -386,7 +386,7 @@
 void flexinit(int argc, char **argv)
 {
     int i, sawcmpflag;
-    char *arg, *flex_gettime(), *mktemp();
+    char *arg, *flex_gettime(), *mktemp(char *);

     printstats = syntaxerror = trace = spprdflt = interactive = caseins = false;
     backtrack_report = performance_report = ddebug = fulltbl = fullspd = false;
--- src/toolsComm/flex/misc.c	2018-09-14 20:44:05.000000000 +0200
+++ src/toolsComm/flex/misc.c	2026-08-24 19:04:13.312251000 +0200
@@ -167,7 +167,7 @@
 /* clower - replace upper-case letter to lower-case
  *
  * synopsis:
- *    Char clower();
+ *    Char clower(int);
  *    int c;
  *    c = clower( c );
  */
--- src/toolsComm/flex/parse.y	2018-09-14 20:44:05.000000000 +0200
+++ src/toolsComm/flex/parse.y	2026-08-24 19:03:23.527093000 +0200
@@ -34,7 +34,7 @@
 
 int pat, scnum, eps, headcnt, trailcnt, anyccl, lastchar, i, actvp, rulelen;
 int trlcontxt, xcluflg, cclsorted, varlength, variable_trail_rule;
-Char clower();
+Char clower(int);
 
 static int madeany = false;  /* whether we've made the '.' character class */
 int previous_continued_action;	/* whether the previous rule's action was '|' */
--- src/toolsComm/flex/sym.c	2018-09-14 20:44:05.000000000 +0200
+++ src/toolsComm/flex/sym.c	2026-08-24 19:12:44.220104000 +0200
@@ -116,7 +116,7 @@
     /* we don't bother checking the return status because we are not called
      * unless the symbol is new
      */
-    Char *copy_unsigned_string();
+    Char *copy_unsigned_string(Char *);
 
     (void) addsym( (char *) copy_unsigned_string( ccltxt ), (char *) 0, cclnum,
 		   ccltab, CCL_HASH_SIZE );
@@ -199,8 +199,8 @@
 
 void ndinstal(char *nd, Char *def)
 {
-    char *copy_string();
-    Char *copy_unsigned_string();
+    char *copy_string(char *);
+    Char *copy_unsigned_string(Char *);
 
     if ( addsym( copy_string( nd ), (char *) copy_unsigned_string( def ), 0,
 		 ndtbl, NAME_TABLE_HASH_SIZE ) )
@@ -235,7 +235,7 @@
 
 void scinstal(char *str, int xcluflg)
 {
-    char *copy_string();
+    char *copy_string(char *);
 
     /* bit of a hack.  We know how the default start-condition is
      * declared, and don't put out a define for it, because it
--- src/toolsComm/flex/scan.c	2018-09-14 20:44:05.000000000 +0200
+++ src/toolsComm/flex/scan.c	2026-08-24 19:20:58.155933000 +0200
@@ -975,7 +975,7 @@
     static int bracelevel, didadef;
     int i, indented_code, checking_used, new_xlation;
     int doing_codeblock = false;
-    Char nmdef[MAXLINE], myesc();
+    Char nmdef[MAXLINE], myesc(Char *);
 
 
     if ( yy_init )
@@ -1497,7 +1497,7 @@
 # line 333 "scan.l"
 {
 			Char *nmdefptr;
-			Char *ndlookup();
+			Char *ndlookup(char *);
 
 			(void) strcpy( nmstr, (char *) yytext );
 			nmstr[yyleng - 1] = '\0';  /* chop trailing brace */

