--- src/util/iocLogServer.c	2018-09-14 20:44:05.000000000 +0200
+++ src/util/iocLogServer.c	2026-08-24 19:52:32.929845000 +0200
@@ -473,14 +473,14 @@
 	 * this task will find out and exit
 	 */
 	{
-		long true = 1;
+		long True = 1;
 
 		status = setsockopt(
 				pclient->insock,
 				SOL_SOCKET,
 				SO_KEEPALIVE,
-				(char *)&true,
-				sizeof(true) );
+				(char *)&True,
+				sizeof(True) );
 		if(status<0){
 			fprintf(stderr, "Keepalive option set failed\n");
 		}
