--- src/libCom/cxxTemplates/tsSLList.h   2018-09-14 20:44:05.000000000 +0200
+++ src/libCom/cxxTemplates/tsSLList.h   2026-08-24 16:20:08.527575000 +0200
@@ -311,13 +311,13 @@
 template < class T >
 inline bool tsSLIterConst<T>::operator == ( const tsSLIterConst<T> &rhs ) const
 {
-    return this->pEntry == rhs.pConstEntry;
+    return this->pEntry == rhs.pEntry;
 }
 
 template < class T >
 inline bool tsSLIterConst<T>::operator != (const tsSLIterConst<T> &rhs) const
 {
-    return this->pEntry != rhs.pConstEntry;
+    return this->pEntry != rhs.pEntry;
 }
 
 template < class T >

